### ISSUE TITLE
Issue on "Credit Card Membership"

### ISSUE THREAD

**Issue by mercurymotos** [twomice/com.joineryhq.percentagepricesetfield#45]
When viewing a contact and selecting the Membership -> "Submit Credit Card Membership" the percentage field displays this : ERROR: percentagepricesetfield_NFMbsQAQVpVZkamC (Hint: Is the Percentage Price Set Field extension enabled?)

Further, if the checkbox is selected, the specified % is not applied (here its 4%) ... instead its a flat $1.

The field works fine on Credit Card Contributions initiated from the member screen and on memberships submitted via the contribution page.
<img width="612" alt="Screen Shot 2022-01-31 at 9 37 10 AM" src="https://user-images.githubusercontent.com/7410246/151844252-50617d63-4e38-46c8-9710-486a1b391134.png">

[EDIT BY @twomice : This problem also applies to the back-office "New Membership" form, accessed e.g. by the "Add Membership" button under a contact's Memberships tab.]

**Comment by twomice** [comment]
Thanks @mercurymotos Can you tell me the version of CiviCRM, CMS, and this extension?

**Comment by mercurymotos** [comment]
doh!
CiviCRM 5.45.1
Drupal 9.3.3
Percentate Price Set Field  1.5.1

**Comment by shaneonabike** [comment]
I also noticed this issue today when attempting to process a transaction. We are using Wordpress in our case. It's more of a backend related issue though.

**Comment by twomice** [comment]
Thanks for reporting this. Oviously I have failed to respond for quite a while. 

In fact I believe that that this extension has never supported proper functionality on back-office forms. I've created a separate issue to address this ('Support proper "additional percentage" functionality on back-office forms' #61), and suggest that it should supercede this issue. So I'll close this ticket and follow up there. Thanks!
